scripts/docker/docker-pull-async.ps1

```diff
@@ -12,5 +12,3 @@ function DockerPull {
   $results
 }
 
-DockerPull microsoft/windowsservercore
```

Comment:
I am using VMWare Fusion 8.5.3 (4696910) on OS/X 10.12.3

My use case is primarily for docker-windows-box use, and Powershell DSC based deployment testing.